extensions/google-translate/src/translate.tsx

```diff
@@ -84,17 +84,10 @@ export default function Command(): ReactElement {
         }
       })
       .catch((error) => {
-        console.error("translate error", error);
-        // HACK: BAD_NETWORK errors are caused by too many requests in quick succession;
-        // ignore and retry – this should be fixed systemically.
-        if (error?.code === "BAD_NETWORK") {
-          setRetryCount(retryCount + 1);
-        } else {
```

Comment:
@fe9lix how this one helped to fix TooManyRequests error? As I understand it is trying to call translate api infinitely while it will not return some result, but in this way it will be just returning TooManyRequests, or is it return success response after a bunch of requests?

Reply:
Anyway, I've changed translate library, so now it should be irrelevant